gnd/src/main/java/com/google/android/gnd/ui/map/MapProvider.java

```diff
@@ -32,36 +25,4 @@
   Fragment getFragment();
 
   Single<MapAdapter> getMapAdapter();
-
-  /**
-   * Interface defining map interactions and events. This a separate class from {@link MapProvider}
-   * so that it can be returned asynchronously by {@link MapProvider#getMapAdapter()} if necessary.
-   */
-  interface MapAdapter {
-
-    Observable<MapMarker> getMarkerClicks();
-
-    Observable<Point> getDragInteractions();
-
-    Observable<Point> getCameraPosition();
-
-    void enable();
-
-    void disable();
-
-    void moveCamera(Point point);
-
-    void moveCamera(Point point, float zoomLevel);
-
-    Point getCenter();
-
-    float getCurrentZoomLevel();
-
-    @SuppressLint("MissingPermission")
-    void enableCurrentLocationIndicator();
-
-    void updateMarkers(ImmutableSet<Feature> features);
-
-    LatLngBounds getViewport();
```

Comment:
The `getViewport` method didn't get ported over to the `MapAdapter`!  I'll need it for offlineareas. It was unused up to this PR tough, so I'm happy to reimplement. Just an FYI.

Reply:
Oops, sorry! Added it back!